### ISSUE TITLE
Pagination with material 1.0.4

### ISSUE THREAD

**Issue by Knoxvillekm** [daniel-nagy/md-data-table#272]
Select in pagination needs to fix.
[Codepen](http://codepen.io/anon/pen/JGBgVp?editors=101)

**Comment by passerbyid** [comment]
I think it's broken because Angular-Material v1.0.4 add `height: $select-label-height` to `.md-select-value *:first-child` in [c19eec4b](https://github.com/angular/material/commit/c19eec4b).

For now, I add this CSS code to fix it.

```css
.md-table-pagination md-select .md-select-value *:first-child {
    height: auto;
}
```

**Comment by daniel-nagy** [comment]
Fixed in **v0.9.10**
